apps/docs/components/edit-feedback.tsx

```diff
@@ -0,0 +1,36 @@
+type Props = {
+  title: string
+  contentPath: string
+}
+
+export default function EditFeedbackLinks({ title, contentPath }: Props) {
+  const pageEditUrl = `https://github.com/IHIutch/draft-ui/tree/main/apps/docs/content/${contentPath}`
+  const pageFeedbackUrl = new URL(
+    `https://github.com/IHIutch/draft-ui/issues/new`,
+  )
+  pageFeedbackUrl.searchParams.set('title', `Feedback for “${title}”`)
+  // Is this the right label to set? Do we need to make a new label for this?
+  pageFeedbackUrl.searchParams.set('labels', 'documentation')
+
+  return (
+    <ul className="mt-3">
+      <li className="py-1">
+        <a
+          className="text-xs text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-white"
```

Comment:
Double-check the link styles from the ToC and these. I'd like them to be as consistent as possible. I don't think you'll be able to copy/paste exactly, because the ToC links have an "active" state, but you should be able to copy over the non-active styles. If might even be more helpful to copy from your Inspector because that would be the outputted classes that the DOM sees, without the `cx` function.

If you're able to do that, I think you'll be able to drop the `py-1` on the list items.

Reply:
Just to clarify what I am looking for: do you want the edit/feedback links to look exactly the same (incl. text size, spacing, colors, hover) or is there a specific inconsistency?

Reply:
Good question. These links the same as the ToC links